ccf_transaction_processor/README.md

```diff
@@ -196,7 +207,7 @@ commit. The ledger authority verifying key can be obtained using the
 `get_ledger_verifying_key` rpc. The verifying key is returned only after
 global commit is successful.
 
-The read-payload-signature feature may be used by PDO clients to
+The `read-payload-signature` feature may be used by PDO clients to
```

Comment:
I may have missed this: Is this feature an RPC, a separate script?

Reply:
The `get_ledger_verifying_key` is an rpc, Regarding `read-payload-signature`, please read it as "signature attached to read-payloads". So no new rpc here. every read rpc response comes with a signature generated by ccf pdo-tp.

Reply:
Got it, thanks!